### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by wbs306**

### CODE REVIEW

File: src/translations/po/zh.po [hunk 1]
```diff
@@ -0,0 +1,245 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# SPDX-FileCopyrightText: 2019, 2020 Vlad Zahorodnii <vladzzag@gmail.com>
```
Comment (zzag) [inline src/translations/po/zh.po:3]: Please remove this line, I'm not the author of this file.

File: src/translations/po/zh.po [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+# wbs <wbs306@gmail.com>, 2021.
```
Comment (zzag) [inline src/translations/po/zh.po:4]:
```suggestion
# SPDX-FileCopyrightText: 2021 wbs <wbs306@gmail.com>
```

**Review by zzag — COMMENTED** [review]
Nice! I have a few license related nitpicks; other than that, it looks really good to me. :-)

File: src/translations/po/zh.po (hunk 2 above)
Comment (wbs306) [reply src/translations/po/zh.po]: Removed.

**Comment by zzag** [comment]
Thanks!

**Pull request merged**